input/fsh/VS_Radiotherapy.fsh

```diff
@@ -1,34 +1,32 @@
-Alias: RO = http://hl7.org/fhir/us/mcode/CodeSystem/mcode-radiotherapy-cs
-
 //----------- MODALITIES -----------------------
 
 ValueSet:  TeleradiotherapyModalityVS
 Id: mcode-teleradiotherapy-modality-vs
 Title: "Teleradiotherapy Modality Value Set"
 Description: "Codes describing the modalities of teleradiotherapy (external beam) procedures. This value set is considered closed (not extensible on ad hoc basis by the implementer)."
-* insert ExpandValueSet
-* RO#PROTON
-* RO#ELECTRON
-* RO#NEUTRON
-* RO#CARBON
-* RO#PHOTON
+// * insert ExpandValueSet
```

Comment:
Do you want to explain why this (and the same line repeated below) is commented out, or omit?

Reply:
So, I was looking forward to when SUSHI will expand the value set and put the rules in place to trigger that in anticipation. However, the rules create the expansion section, so IGP would pass over the VS and not expand it. When the current SUSHI sprint is complete, I will reintroduce these statements. So I left them there so it will be easier to figure out later.